Documentation/articles/getting_started/1_creating_a_new_project_vs.md

```diff
@@ -1,19 +1,75 @@
 # Creating a Project with Visual Studio
 
-Start up Visual Studio and select **New Project...** in the upper left corner.
+MonoGame is available on Windows for development using the instructions below to get started with MonoGame 3.8.
+
+> Note, MonoGame 3.8 projects are no longer compatible with earlier versions of MonoGame.  If you wish to work on or build older MonoGame projects, then you will still need to install [MonoGame 3.7.1](https://www.monogame.net/downloads/) or earlier to open them.
+
+## Prerequisites
+
+You will need a copy of [Visual Studio 2019](https://www.monogame.net/downloads/) or later installed (any edition, including Community) before installing MonoGame, with the following components (depending on your target platform):
+
+![Visual Studio optional components](~/images/getting_started/1_installer_vs_components.png)
+
+* .Net Desktop Development - For Windows GL / DX platforms
+* Mobile Development with .NET - For Android / iOS platforms
+* Universal Windows Platform development - For Windows 10 / Xbox UWP
+
+When installing Visual Studio, it also is recommended to include the "**.NET Core cross-platform development" components:
+
+![DotNet Core component](~/images/getting_started/1_netcorecomponet.png)
+
+> Alternatively, you can specifically install the [.NET Core SDK from here](https://dotnet.microsoft.com/download), SDK Versions 3.1 and above.
+
+## Install MonoGame Templates
+
+From MonoGame 3.8, we no longer use MSI Installers for deploying the framework and templates as this creates hard dependencies to specific machine setups or DLL locations.  Instead, MonoGame now uses NuGet and the DotNet deployment platform to reference a specific version of MonoGame online.
```

Comment:
This information is on the migration page. IMO we shouldn't mention the old setup in any other page. It would be better to link the migration page if we think that's necessary.